### ISSUE TITLE
wordpress not reading css from child theme.

### ISSUE THREAD

**Issue by Hammer1980** [understrap/understrap-child#389]
I have set up the understrap starter child theme  downloading it from the website, and whilst it reads changes in template files, like header.php etc, it just won't make the changes in style.css.

Any ideas what could be wrong, or is there something I should alter ?

Thank you.

**Comment by wpbuilds** [comment]
Hi @Hammer1980 , have you read the documentation on how to use the child-theme effectively? - `https://docs.understrap.com/#/`

**Comment by Hammer1980** [comment]
Yes. First place I looked. Since my last post I saw of a facebook group said about editing the chlid-theme. min css.  I did that by renaming the child-theme.css to the min version and that seems to work.

Is there a better way. The bits I saw on the link you sent me has very little about the child theme that I can find and the read me with it doesn't explain either, unless I missing something blatantly obvious, which I wouldn't rule out by the way.  Clearly by your post you think I haven't !  I wouldn't be posting here if I hadn't exhausted every avenue I could find :) 

Only found this by chance, wish I had found this theme and system a while ago.

**Comment by wpbuilds** [comment]
If you have followed ALL of the installation and setup steps as per the documentation - You should not be having to change the naming conventions of CSS files for this child-theme to work out of the box. The child-theme uses [SASS](https://sass-lang.com/) to compile to minified CSS files via [Node JS](https://nodejs.org/en) commands. Note that without "context" to your specific issue it is very hard  for anyone here to help troubleshoot you particular problem.

**Comment by Hammer1980** [comment]
I am doing this on a live domain. I am probably a bit out of my depth here to be honest. I have just uploaded the theme, and the child theme. 

I totally agree abut the context bit. I was assuming it was something I could do in the functions file and I was expecting, well more hoping it would be,' you haven't done that line' or 'change this'.

I need to get something online quick for someone as their current them is not php8.2 compliant, currently 7.4.  A quick and dirty name change will have to do for now, but when I read up about the development course I think I really fancy having a crack at that.

Really grateful for your time in trying to get me sorted.

**Comment by Hammer1980** [comment]
Now for some reason, today my browser developer tools has stopped showing me the line numbers of css ! Handy with 15k lines of css :)
